pickem/pickem_homepage/views.py

```
@@ -4041,6 +4076,248 @@ def serialize_comment(comment):
+                'content': post.content,
+            }
+        })
+
+    except Http404:
+        return message_board_not_found()
+    except json.JSONDecodeError:
+        return JsonResponse({
+            'success': False,
+            'error': 'Invalid JSON data'
+        }, status=400)
+    except Exception as e:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)